cmd/korg/remove.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func RemoveMemberFromOrgs(o Options, username string) error {
+	if invalidOrgs := findInvalidOrgs(o.Orgs); len(invalidOrgs) > 0 {
+		return fmt.Errorf("specified invalid orgs: %s", strings.Join(invalidOrgs, ", "))
+	}
+
+	if !o.Confirm {
+		fmt.Println("!!! running in dry-run mode. pass --confirm to persist changes.")
+	}
+
+	configsModified := []string{}
+	for _, org := range o.Orgs {
+		fmt.Printf("removing %s from %s org\n", username, org)
+
+		relativeConfigPath := fmt.Sprintf(orgConfigPathFormat, org)
+		configPath := filepath.Join(o.RepoRoot, relativeConfigPath)
+
+		config, err := readConfig(configPath)
+		if err != nil {
+			return fmt.Errorf("reading config: %s", err)
+		}
+
+		if !stringInSliceCaseAgnostic(config.Members, username) {
+			return fmt.Errorf("user %s doesn't exist in org %s", username, org)
+		}
+
```

Comment:
In case of admins, let's call out the username is an admin
(currently we get `Error: user <username> doesn't exist in org kubernetes`)

```suggestion
                if stringInSliceCaseAgnostic(config.Admins, username) {
                        return fmt.Errorf("user %s is an admin for org %s", username, org)
                }
                
		if !stringInSliceCaseAgnostic(config.Members, username) {
			return fmt.Errorf("user %s doesn't exist in org %s", username, org)
		}

```

Reply:
This is a good suggestion. Added.